### PR TITLE
Specter.ice_connection_state/2

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

**Comment by sax** [comment]
This is awesome! I'll probably merge this as-is, but wondered if you had any thoughts about long-term management of atoms. 

I know there's a way to define an enum to automatically encode to atoms, but because this enum doesn't belong to Specter I'm guessing we can't do that. I was considering shadowing the webrtc.rs enums and implementing the From trait... that would move some of this code out of the receive loop, but I can't decide if it's better or worse.

**Comment by sax** [comment]
Looks like there's a @doc instead of a @typedoc. I'll be at a computer later today and can fix that if you can't get to it first.

File: lib/specter.ex [hunk 1]
```diff
@@ -179,6 +179,22 @@ defmodule Specter do
   """
   @type session_description_t() :: String.t()
 
+  @doc """
```
Comment (sax) [inline lib/specter.ex:182]:
```suggestion
  @typedoc """
```

**Review by sax — APPROVED** [review]
Fix typedoc

**Comment by sax** [comment]
...I might have gotten it from my phone. Anyways, let me know what you think about the enum idea!

**Comment by sax** [comment]
Take a look at #2. I'm going to read up on it a little more, but think that might be a good way to handle things like this.

**Comment by mickel8** [comment]
Wow, that was really quick. I was going to improve the code a little bit but I wanted to create PR to let you know I started implementing a new function so that we don't duplicate code :smiley:

**Comment by sax** [comment]
That sounds great. Yeah, I probably won't be able to add much until the middle of next week, so chances are you'll get there ahead of me.

One note is that I was considering putting the other crate peer connection enums into `peer_connection/peer_conn_state.rs`.

**Pull request merged**